### ISSUE TITLE
bug(Tabs): Accessibility issues with Tab Pagination

### ISSUE THREAD

**Issue by drc-nloftsgard** [angular/components#32856]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

The information below is from a report from Evinced with 2 findings (Interactable Role and Accessible Name) on the Tab Pagination elements:

### Interactable Role

Interactive elements, such as buttons, links, and inputs, must convey that they can be interacted with by using the proper semantics.

**Severity**: Critical
**Cause**: Screen readers are unable to identify that the element is interactive due to missing or incorrect semantics.
**Effect**: Screen reader users may not find the content or may not realize that the the content is interactive.
[Read more in Evinced Knowledge Base](https://knowledge.evinced.com/system-validations/interactable-role)
**How To Fix**:
Use native HTML interactive elements such as &lt;button&gt;, &lt;a&gt;, &lt;select&gt;, &lt;input&gt;. Alternatively, add the appropriate ARIA role that corresponds to the content purpose.

WCAG Violations: [4.1.2 Name, Role, Value (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)
Section 508: 1194.21(d)
EN 301 549: 9.4.1.2 Name, Role, Value

### Accessible Name

Elements that can be interacted with, such as buttons, links, and input fields, must have an accessible name. The name should indicate which information or action is required, and/or what will happen if they interact with it.

**Severity:** Critical
**Cause:** The element does not have an accessible name to inform the screen reader of its purpose and function.
**Effect**: Assistive technologies use the accessible name to label, announce and trigger the element's action. If there's no valid accessible name, assistive technologies won't be able to understand the element correctly.
[Read more in Evinced Knowledge Base](https://knowledge.evinced.com/system-validations/accessible-name)
**How To Fix**:
Depending on the type of element, either add a descriptive text to the body of the tag or add the appropriate attributes to the tag.

WCAG Violations: [4.1.2 Name, Role, Value (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

### Reproduction

This issue is visible on the Material documentation page:
https://material.angular.dev/components/tabs/overview

Resize your window to 700px or less for the Tab Pagination 'chevron' elements to appear. Inspect these elements to observe their (lack of) accessible attributes.

### Expected Behavior

These elements should be hidden from keyboard and screen readers.
These elements should be labeled for voice navigation.

The potential fix could include applying the following attributes to each arrow

`role=button aria-hidden=”true” tabindex=-1`

`aria-label` with 'Scroll Tabs Right' or 'Scroll Tabs Left'

### Actual Behavior

Each tab pagination 'button' is composed of 2 `div` elements:

```
<div mat-ripple="" class="mat-ripple mat-mdc-tab-header-pagination mat-mdc-tab-header-pagination-after">
    <div class="mat-mdc-tab-header-pagination-chevron"></div>
</div>
```
This is an interactive element that isn't adequately labeled / accessible for voice navigation users.

### Environment

- Angular: 20
- CDK/Material: 20
- Browser(s): Chrome, any
- Operating System (e.g. Windows, macOS, Ubuntu): any

**Comment by crisbeto** [comment]
We intentionally don't apply any roles to the buttons since they aren't necessary. A user navigating with the keyboard can reach the off-screen tabs using the left/right arrow buttons directly.

**Comment by drc-nloftsgard** [comment]
> We intentionally don't apply any roles to the buttons since they aren't necessary. A user navigating with the keyboard can reach the off-screen tabs using the left/right arrow buttons directly.

Thanks for the reply. This is true that keyboard users can navigate fine. It is not a good experience for voice navigation users, they would have to use a mouse grid to navigate which is tedious. Adding appropriate attributes to these chevron arrows would greatly improve the experience for voice navigation users.
